src/compose/ComposeMenu.js

```diff
@@ -19,6 +19,26 @@ type Props = {
   onExpandContract: () => void,
 };
 
+/**
+ * In 2017, Apple adopted a new standard image format, HEIF, which stands
+ * for High Efficiency File Format; Apple photos are named with the '.heic'
+ * extension. When an image file with the extension '.heic' is uploaded to the
+ * server,the image becomes unviewable within a message list. Interestingly,
+ * the `file` command line utility reports that images saved under this format
+ * are JPEG files, so there's likely some pecularity with how the server handles
+ * file uploads based on the extension.
+ *
+ * If an image file name has the '.heic' extension, and the uri of the image on a
+ * mobile device has an extension for the JPEG format, fixFileNameFromUri replaces
+ * that extension with '.jpeg'.
+ */
+export const fixFileNameFromUri = (uri: string, fileName: string): string => {
+  if (/\.jpe?g$/i.test(uri)) {
```

Comment:
Should we do an `if` for `Platform.OS === 'ios'`?
This might never be `true` on Android, but this `if` will make the behavior more exact to our intentions.

Reply:
The function doesn't depend on anything particular to iOS to run, so such a check would be unnecessary. In the case that a user on Android attempts to upload a HEIC file (though, I'm not sure if that's even possible), the logic will be used.

Reply:
According to Wikipedia, [HEIF is supported by Android Pie](https://en.wikipedia.org/wiki/High_Efficiency_Image_File_Format#Support).